src/ProjectOrigin.WalletSystem.Server/CommandHandlers/TransferCertificateCommandHandler.cs

```diff
@@ -133,7 +126,7 @@ private async Task<IEnumerable<Slice>> ReserveRequiredSlices(IEnumerable<Slice>
         }
         _unitOfWork.Commit();
 
-        _logger.LogTrace($"{takenSlices.Count} slices reserved.");
+        _logger.LogTrace("{count} slices reserved.", takenSlices.Count);
```

Comment:
Explain this difference again :)

Reply:
There is also an example here [example](https://serilog.net)

old version AND new version in text format output:

```log
08:55:22 [Trace] 1 slices reserved.
```

but in json output the format would be similar to 

## OLD
```json
{
    "Timestamp": "2023-09-28T06:55:22.9438887+00:00",
    "Level": "Trace",
    "MessageTemplate": "1 slices reserved.",
    "Properties": {
        ...
    }
}
```

## New 
```json
{
    "Timestamp": "2023-09-28T06:55:22.9438887+00:00",
    "Level": "Trace",
    "MessageTemplate": "{count} slices reserved.",
    "Properties": {
        "count": "1",
        ...
    }
}
```